app/src/main/java/com/andela/helpmebuy/activities/PurchaseHistoryActivity.java

```diff
@@ -46,12 +46,13 @@ public void intializeView() {
     }
 
     public void fetchPurchaseRequest() {
-        new FirebaseCollection<>(Constants.PURCHASE_REQUEST + "/" + userId, PurchaseRequest.class).getAll(new DataCallback<List<PurchaseRequest>>() {
+        new FirebaseCollection<>(Constants.PURCHASE_REQUEST + "/" + userId, PurchaseRequest.class)
+                .getAll(new DataCallback<List<PurchaseRequest>>() {
```

Comment:
Not sure if i got this right, but would this still work? the `.getAll` is on a new line, shouldn't it be above?